### PR TITLE
Add testing farm initial documentation

### PR DESCRIPTION
**Pull request by thrix**
This PR adds documentation about Testing Farm, Packit's testing service.

The testing is linked from Packit Service documentation, which is the
only supported way of testing for now.

Signed-off-by: Miroslav Vadkerti <mvadkert@redhat.com>

### CODE REVIEW

**Comment by thrix** [comment]
@psss fyi

**Comment by thrix** [comment]
@TomasTomecek heya, could somebody look at this piece of docs documenting initial version of Testing Farm? Ty

File: content/testing-farm/_index.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+The meaning of individual attributes is as follows:
+
+* Summary — an optional but useful attribute describing high-level purpose of the testset.
+* Discover — instructs to fetch tests from given repository and select relevant ones by provided `filter`.
+* Prepare — specifies which ansible playbook should be applied to prepare environment for testing.
+* Execute — defines that the beakerlib framework should be used for running the tests.
+
+## Issues & RFEs
+
+If you have found an issue or have an RFE, please use Packit Service's Github issue tracker to [file an issue][issues].
```
Comment (rpitonak) [inline content/testing-farm/_index.md]: We should probably discuss this. Don't you want to have your own issue tracker?

Comment (psss) [reply content/testing-farm/_index.md]: I'm ok with tracking testing farm issues in a separate project. But this is rather question for @thrix.

**Comment by psss** [comment]
I've added a couple of real-life examples from `fmf` as discussed with @rpitonak in chat.

File: content/testing-farm/_index.md (hunk 1 above)
Comment (thrix) [reply content/testing-farm/_index.md]: changed it

**Comment by psss** [comment]
It looks fine, @thrix.

**Comment by rpitonak** [comment]
Looks good to me, feel free to merge and update later if needed.

**Comment by TomasTomecek** [comment]
since it's rendered, it won't display on the website; let me fix

**Comment by thrix** [comment]
A stupid typo slipped through ... :(

**Pull request merged**